### DISCUSSION TITLE
关于skywalking-go引入后在macos编译出错的问题

### DISCUSSION THREAD

**Discussion by NecoLiang** [apache/skywalking#11375] [Q&A] [answered]
根据官方文档引入"github.com/apache/skywalking-go"后，在执行go build 命令时，出现报错：
/github.com/prometheus/client_golang@v1.12.2/prometheus/process_collector_other.go:20:2: zip: not a valid zip file
/github.com/spf13/viper@v1.16.0/internal/encoding/toml/codec.go:4:2: zip: not a valid zip file
/github.com/aliyun/alibaba-cloud-sdk-go@v1.61.1704/sdk/errors/server_error.go:21:2: zip: not a valid zip file
希望后续有解决可以得到回复

**Comment by wu-sheng** [comment, marked as answer]
No Chinese on Github. And why you are so impolite to copy your question everywhere?

**Comment by wu-sheng** [comment]
Super annoying.
